documents/knowledge/elements.yaml

```diff
@@ -2706,6 +2706,12 @@
   arity: 0
   description: Rotate Stack Right
   vectorise: false
+  
+- element: "🍪"
+  name: Cookie
+  description: print "cookie" forever
+  vectorise: false
+  arity: 0
```

Comment:
Arity is 1

Reply:
No, arity is 0. It takes no arguments.

Reply:
But it still consumes one in the code

Reply:
Well, it should take no arguments, anyway. I guess it does have arity 1.

Reply:
> But it still consumes one in the code

Doesn't matter since it effectively overrides *everything* else